source/nodejs/adaptivecards/src/strings.ts

```diff
@@ -34,4 +34,7 @@ export class Strings {
     static readonly hints = {
         dontUseWeightedAndStrecthedColumnsInSameSet: () => "It is not recommended to use weighted and stretched columns in the same ColumnSet, because in such a situation stretched columns will always get the minimum amount of space."
     }
+    static readonly fallback = {
```

Comment:
I think using the terms "defaults" for the static member and "inlineActionTitle" for the actual resource function might be better.

Reply:
Renamed